spec/dummy/config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  get "success" => "test_responses#success"
+  get "server-error" => "test_responses#server_error"
+  get "client-error" => "test_responses#client_error"
```

Comment:
do you think we should test the integration with `ActionDispatch::ShowExceptions` middleware? where we redirect the errors to use a common error page?  I'm talking about this block https://github.com/rails/rails/blob/3872bc0e54d32e8bf3a6299b0bfe173d94b072fc/actionpack/lib/action_dispatch/middleware/show_exceptions.rb#L36-L48 WDYT?



```suggestion
  get "client-error" => "test_responses#client_error"
  get "not_found" => "test_responses#client_error"
  match "404", to: "errors#not_found", code: "404", via: :all
```

Reply:
Is this what you had in mind?

https://github.com/errm/promenade/pull/21/files#diff-7a6d85d77b7198cb9da2de4c0ae1fe237c3ee160a8267ec185e8bb1c5d5286d1R68-R82